akka-persistence-typed/src/main/scala/akka/persistence/typed/WallClock.scala

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.typed
+
+/**
+ * A time source.
+ */
+trait WallClock {
+  def currentTimeMillis(): Long
+}
+
+object WallClock {
+
+  /**
+   * Always increasing time source.
+   */
+  val AlwaysIncreasingClock: WallClock = new internal.AlwaysIncreasingClock()
+}
```

Comment:
Could this be general enough to go in `akka.util` perhaps?

Reply:
Can't hurt